science/gcal/files/patch-trap-remove-may-month-check-fix.diff

```diff
@@ -0,0 +1,68 @@
+--- src/utils.c.orig	2020-11-27 23:18:52.000000000 +1300
++++ src/utils.c	2020-11-27 23:28:24.000000000 +1300
+@@ -1337,35 +1337,36 @@
+   };
+ 
+ 
+-  if (month == 5)
+-    {
+-      static char *mayname;
+-      static Bool is_initialized = FALSE;
+-
+-
+-      /*
+-         Now let's remove the trailing BLANK character
+-         from the complete text for the month MAY.
+-       */
+-      if (!is_initialized)
+-	{
+-	  register int len;
+-	  auto char *s = _(name[month]);
+-
+-
+-	  len = (int) strlen (s) - 1;
+-	  mayname = (char *) my_malloc (len,
```

Comment:
It's odd that allocated memory is 2 bytes less than string length (including trailing zero), but then the full string is copied to allocated memory. Maybe len should be strlen (s) + 1 instead.

Reply:
I am not sure. I have removed commented out code and added a statement to return "May" when appropriate. A lot simpler.